docs/api/secret.md

```diff
@@ -198,6 +199,34 @@ Updates parameters of an existing secret.
     }
     ```
 
+<a name="example-update-single-value-secret"/>
+
+### Example: Update Single Value Secret
+
+You can update a single value as a secret on Concord as follows:
+
+* **URI** `/api/v1/org/${orgName}/secret/${secretName}`
```

Comment:
Change the example to use curl for consistency with the other examples in this document?

Reply:
yep